cypress-tests/cypress/support/commands.js

```diff
@@ -376,22 +515,30 @@ Cypress.Commands.add("syncRefundCallTest", (det, globalState) => {
   const refundId = globalState.get("refundId");
   cy.request({
     method: "GET",
-    url: `${globalState.get("baseUrl")}/refunds/${refundId}?force_sync=true`,
+    url: `${globalState.get("baseUrl")}/refunds/${refundId}`,
     headers: {
       "Content-Type": "application/json",
       "api-key": globalState.get("apiKey"),
     },
   }).then((response) => {
+    const xRequestId = response.headers['x-request-id'];
+    if (xRequestId) {
+      cy.task('cli_log', "x-request-id ->> " + xRequestId);
+    } else {
+      cy.task('cli_log', "x-request-id is not available in the response headers");
+    }
```

Comment:
Since this piece of code is repeated everywhere, can we create a function out of it and call where ever necessary just to reduce code duplication?

Something like:

```js
// function call
x_request_logger(response.headers['x-request-id']);

// define the function something like the below given pseudo code?

function x_request_logger(xRequestId: String) {
    if (xRequestId) {
      cy.task('cli_log', "x-request-id ->> " + xRequestId);
    } else {
      cy.task('cli_log', "x-request-id is not available in the response headers");
    }
}
```